### PR TITLE
Update description

### PR DESCRIPTION
**Pull request by willlam**
Updating description of Taplytics and its offerings

### CODE REVIEW

**Review by alexs-mparticle — APPROVED** [review]

**Comment by suemparticle** [comment]
Hi @willlam thank you for opening this pull request. Please sign the contributor's agreement and we can merge this in for you.

File: src/pages/integrations/taplytics/event/index.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 title: Event
 ---
 
-[Taplytics](https://taplytics.com/) is an intelligent customer experience cloud specializing in turning first-time users into lifetime customers through magical moments.
+[Taplytics](https://taplytics.com/) is an A/B Testing, Feature Management and Behavioral Messaging platform that helps companies turn first-time users into lifetime customers.
 
-The Taplytics integration is an embedded kit integration and supports the Android, iOS, tvOS, and Web platforms.
+The Taplytics integration is an embedded kit integration and supports the Android, AndroidTV/FireTV, iOS, tvOS, and Web platforms.
```
Comment (MystiBerry) [inline src/pages/integrations/taplytics/event/index.md:7]:
```suggestion
The Taplytics integration is an embedded kit integration and supports the Android, iOS, tvOS, and Web platforms.
```

**Comment by dmiller-mp** [comment]
hi @willlam looking at the supported inputs for Taplytics - https://app.mparticle.com/directory/129 - only iOS/Android and Web are shown - should connections be supported from other inputs
Does the Taplytics Android kit support both Android and Fire TV?  
Does the iOS kit support both iOS and tvOS?

**Comment by MystiBerry** [comment]
Feel free to reopen if William signs the CLA.

**Pull request closed without merging**